spec/poros/cast_member_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe CastMember do
```

Comment:
We could potentially use VCR here but creating the object in the test also works.

Edit: discussed on pair.